### ISSUE TITLE
elpy-config got different behaviors on different machines with the same elpy version

### ISSUE THREAD

**Issue by blldw** [jorgenschaefer/elpy#1926]
# Steps to reproduce

`M-x elpy-config` on two different machines got different results as follows:
Machine-1:
Emacs.............: 28.0.50
Elpy..............: 1.35.0
Virtualenv........: None
Interactive Python: python3 3.8.10 (/home/jason/.conda/envs/py38_pt18/bin/python3)
RPC virtualenv....: py38_pt18 (/home/jason/.conda/envs/py38_pt18)
 Python...........: python3 3.8.10 (/home/jason/.conda/envs/py38_pt18/bin/python3)
 Jedi.............: 0.18.0
 Autopep8.........: Not found
 Yapf.............: Not found
 Black............: Not found
Syntax checker....: Not found (flake8)

Machine-2:
Emacs.............: 28.0.50                                                     
Elpy..............: 1.35.0                                                      
Virtualenv........: None                                                        
Interactive Python: python3 3.8.10 (/home/aero/.conda/envs/py38_pt18/bin/python3)
RPC virtualenv....: py38_pt18 (/home/aero/.conda/envs/py38_pt18)                
Python...........: python3 3.8.10 (/home/aero/.conda/envs/py38_pt18/bin/python3)
Jedi.............: 0.18.0                                                      
Rope.............: Not found   # Pay an attention on this line.                                                  
Autopep8.........: Not found                                                   
Yapf.............: Not found                                                   
Black............: Not found                                                   
Syntax checker....: Not found (flake8)     

Both machines run Ubuntu 20.04 OS, with the same Emacs version and configuration (including elpy configuration) and I wonder the reason the different result got by the same elpy version.

This different behavior confuses me, since I got a 100% CPU usage on Machine-1 on editing A.py while elpy works perfectly on Machine-2 on editing just the same A.py under the same project root. More specificly, the process leading to 100% CPU usage is:
/home/jason/.conda/envs/py38_pt18/bin/python3 -W ignore -m elpy.__main__

Thanks for any tips.

**Comment by blldw** [comment]
I found the slight differences in elpy version on Machine-1 and Machine-2 and made a mistake, so the problem "elpy-config behaves differently on different machine with the same elpy version" disappears. 

However, the problem about 100% CPU usage still exists. I also noticed previous posts, and the problem seems not solved completely, so the way you soved this problem is welcome.

**Comment by blldw** [comment]
The problem of 100% CPU usage disappeared, and it does not matter with ELPY! A folder named "data" in my project, and ELPY will scan and parse files in that folder. However, the "data" folder is huge (some 75 GB), including many subfolders and files, and scanning and parsing consume much time. I tried to add that folder to `elpy-project-ignored-directories,` but it seemed not to work, so I moved the folder somewhere, just out of my project root directory, and it worked, no more 100% CPU usage.
